### DISCUSSION TITLE
Error: 'email' does not exist in type 'UserWhereUniqueInput'

### DISCUSSION THREAD

**Discussion by reynro89** [prisma/prisma#12561] [Q&A]
I keep getting the "does not exist in type UserWhereUniqueInput error for a 'findUnique' search in VSCode.  "@unique" is working perfectly for that object, and this has seemingly 0 impact on the endpoint as it still finds a unique row based on the email address. I just don't really want that error showing, and if it is impacting in some way want it to not do that.

```
model User {
  id        Int      @id @default(autoincrement())
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt

  email    String @unique
  password String

  firstName String?
  lastName  String?

  bookmarks Bookmark[]

  @@map("users")
}
```

```
    async login (dto: AuthDto) {
        const user =
        await this.prisma.user.findUnique({
          where: {
            email: dto.email,
          },
        });
    }
```

**Comment by austincrim** [comment]
Hey @reynro89 👋 ,

If you have re-ran `prisma generate` and the error still shows, sometimes you need to restart the TypeScript server to pick up the latest changes.

In VSCode, open Command Palette (<kbd>CMD/CTRL + Shift + P</kbd>) -> `TypeScript: Restart TS Server`

**Reply by pmf-dave** [reply]
@austincrim Thank you! This did it for me! Error was bothering me for about an hour before I found this.

**Reply by maidieu999** [reply]
@austincrim  Thank you, bros. That works for me

**Reply by fervargasuv** [reply]
Literally that solve me the same problem thx!!!

**Reply by lawrencemauku** [reply]
you can also restart both prisa server and typescript server if the error does not get resolved by restarting only one.

**Reply by 0xanonymeow** [reply]
This works! Thanks. 😃

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
